packages/connections/src/XHRConnection.js

```diff
@@ -0,0 +1,170 @@
+import AbstractConnection from "./AbstractConnection";
+import ConnectionEvent from "./ConnectionEvent";
+
+export const ALLOWED_METHODS = ["GET", "POST", "PUT", "DELETE"];
+
+/**
+ * Basic XHR Connection
+ * @todo add/remove headers
+ */
+export default class XHRConnection extends AbstractConnection {
+  /**
+   * Initializes an Instance of XHRConnection
+   * @constructor
+   * @param {string} url – URL to be fetched
+   * @param {object} [options]
+   * @param {string} [options.method=GET] – used method
+   * @param {*} [options.body] – payload for request
+   * @param {object} [options.headers] – additional headers (like content-type)
+   * @param {string} [options.responseType]
+   */
+  constructor(url, options = {}) {
+    super(url);
+
+    const {
+      method = "GET",
+      body = null,
+      headers = {},
+      responseType = "json"
```

Comment:
Shouldn't that be `application/json`?

Reply:
Nope, it's just `json`... https://github.com/dcos/dcos-ui/blob/898438d8a6ce629d2743672a77bc31fb81b92f88/packages/connections/README.md#options-optional